src/main/resources/io/jenkins/plugins/coverage/CoveragePublisher/config.jelly

```diff
@@ -4,4 +4,61 @@
         <f:hetero-list name="adapters" descriptors="${descriptor.listCoverageReportAdapterDescriptors}"
                        items="${instance.adapters}" hasHeader="true"/>
     </f:block>
+
+    <f:advanced>
```

Comment:
copied code as well, right?

Reply:
Yeah, I copy it from Cobertura plugin. Should I add the license in Jelly file?